rpm/calico.spec

```diff
@@ -2,8 +2,8 @@
 
 Name:           calico
 Summary:        Project Calico virtual networking for cloud data centers
-Version:        1.3.0
-Release:        0.3%{?dist}
+Version:        1.3.0~pre.2
```

Comment:
This is wrong.  RHEL pre-release versions should look like Version: 1.3.0, Release: 0.2.pre%{?dist}.  Increment the 2 for each build.